postgresmodels/pagination.go

```diff
@@ -0,0 +1,15 @@
+package postgresmodels
+
+type BaseReport struct {
+	// TotalChunksExpected and TotalChunksRecieved are used to track the progress of the report.
+
+	// Total number of chunks expected. Will be populated with the ReportNumber of the LastReport (IsLastReport == true)
```

Comment:
Maybe we should set it to lastreport+1. since this is not "last expected chunk", but "total chunks expected".
WDYT?

Reply:
@Bezbran 
yeah, we start counting at 0 so we have to do it as you suggested.